docs/teaching.rst

```diff
@@ -0,0 +1,24 @@
+.. -*- mode: rst -*-
+
+Yellowbrick for Teachers
+========================
+
+For teachers and students of machine learning, Yellowbrick can be used as a framework for teaching and understanding a large variety of algorithms and methods. In fact, Yellowbrick grew out of the teaching data science courses at Georgetown's School of Continuing Studies!
```

Comment:
Is there a way to reword the second sentence in this paragraph:

> In fact, Yellowbrick grew out of the teaching data science courses at Georgetown's School of Continuing Studies!

Or perhaps remove the word "the"? I just found that part a little unclear, but it may just be the way I'm reading it.

Reply:
Done!